odl/tomo/backends/astra_cuda.py

```diff
@@ -224,7 +234,9 @@ def call_backward(self, proj_data, out=None):
         if self.geometry.ndim == 2:
             astra.data2d.store(self.sino_id, proj_data.asarray())
         elif self.geometry.ndim == 3:
-            swapped_proj_data = np.rollaxis(proj_data.asarray(), 2, 0)
+            shape = (-1,) + self.geometry.det_partition.shape
+            reshaped_proj_data = proj_data.asarray().reshape(shape)
+            swapped_proj_data = np.rollaxis(reshaped_proj_data, 2, 0)
```

Comment:
Are we sure there is no performance regression from these changes? I'd like an informal benchmark.